src/me/ryanhamshire/AutomaticInventory/AIEventHandler.java

```diff
@@ -210,12 +211,46 @@ public void run()
             if(currentStack != null) return;
 
             ItemStack bestMatchStack = null;
+	    ItemStack bestShulkerStack = null;
             int bestMatchSlot = -1;
+	    int bestMatchShulkerSlot = -1;
             int bestMatchStackSize = Integer.MAX_VALUE;
+	    boolean bestMatchShulker = false;
```

Comment:
indents look... off. Are you using tabs perhaps?

Reply:
Forgot to change the default in IDE... will fix